install-wings.sh

```diff
@@ -41,17 +45,9 @@ if ! [ -x "$(command -v curl)" ]; then
   exit 1
 fi
 
-# define version using information from GitHub
-get_latest_release() {
-  curl --silent "https://api.github.com/repos/$1/releases/latest" | # Get latest release from GitHub api
-    grep '"tag_name":' |                                            # Get tag line
-    sed -E 's/.*"([^"]+)".*/\1/'                                    # Pluck JSON value
-}
-
-echo "* Retrieving release information.."
```

Comment:
This message is not re-added down below along with the `get_latest_release` function. Do you think it is unnecessary to retain the info message? Retrieving information about the latest release can sometimes take a few seconds, especially when the GitHub API is unresponsive which is why it was there in the first place. But I am open to changes, of course, I'd just like to hear your reasoning behind the change and your point of view.

Reply:
I think we can remove the `get_latest_version` entirely, but that's just my opinion. I think if we show latest instead of version number it would be just as good. So should I add it back or remove?

Reply:
I like the fact that it displays the version, and not just `latest` because it is so vague. I reckon you should add the `* Retrieving release information..` message back if the version fetching is going to be kept, which I think it should, does it sound reasonable?

Reply:
Ok it's back...